subsys/usb/class/netusb/function_ecm.c

```diff
@@ -236,15 +236,7 @@ int ecm_send(struct net_pkt *pkt)
 
 	net_hexdump_frags("<", pkt);
 
-	if (!pkt->frags) {
-		remaining = append_bytes(send_buf, sizeof(send_buf),
-					 net_pkt_ll(pkt),
-					 net_pkt_ll_reserve(pkt),
-					 remaining);
-		if (remaining < 0) {
-			return remaining;
-		}
-	} else {
+	if (pkt->frags) {
```

Comment:
there is already a fix for it, #4641

Reply:
Thanks for noticing. Repushed with that commit outside the PR.